### ISSUE TITLE
Request bodies should be marked as required

### ISSUE THREAD

**Issue by OlegYch** [endpoints4s/endpoints4s#1280]
using openapi module v4.5.1 i get a request descriprion like this:
```
        "requestBody": {
          "content": {
            "application/json": {
              "schema": {
                "$ref": "#/components/schemas/CompleteRequest"
              }
            }
          }
        },
```
and it's missing `"required": true` field which means someone could call the endpoint without body

**Comment by julienrf** [comment]
I agree this should be fixed. We should make sure `"required": true` is not set when we use `emptyRequest`, though. Would you be interested in submitting a PR for this? I can guide you and indicate where are the files to change.

**Comment by OlegYch** [comment]
@julienrf seems easy enough, submitted
